Sources/Extensions/Foundation/ArrayExtensions.swift

```diff
@@ -280,6 +280,23 @@ public extension Array {
             value += slice
         }
     }
+    
+    
+    /// SwifterSwift: Returns an array of slices of lenght "size" from the array.  If array can't be split evenly, the final slice will be the remaining elements.
+    ///
+    /// - Parameters:
+    ///   - size: The size of the slices to be returned.
+    public func split(size: Int) -> [ArraySlice<Element>]? {
```

Comment:
@LucianoPAlmeida I'm also thinking this method may be better named as `group(by size: Int)`. I vaguely remember discussing this when adding the `groupByKey` extension recently.